homeassistant/helpers/config_validation.py

```diff
@@ -847,11 +847,15 @@ def custom_serializer(schema: Any) -> Any:
 PLATFORM_SCHEMA_BASE = PLATFORM_SCHEMA.extend({}, extra=vol.ALLOW_EXTRA)
 
 ENTITY_SERVICE_FIELDS = {
-    vol.Optional(ATTR_ENTITY_ID): comp_entity_ids,
+    vol.Optional(ATTR_ENTITY_ID): vol.Any(
+        comp_entity_ids, dynamic_template, vol.All(list, template_complex)
```

Comment:
I realize this is a bit of a weird one.

In order:
- Is it just a single or list of ordinary entity_ids?
- Is it a single template string?
- Or is a list of entity IDs (e.g., mixed templates with static entity_id strings)

Alternatively, we could just handle complex only here, but that would degrade config validation.

Reply:
Let's add this as a comment to the code.

Reply:
Comment added